.github/CODEOWNERS

```diff
@@ -100,6 +100,8 @@ packages/react-components/ @layershifter @miroslavstastny
 packages/react-focus/src/components/FocusZone/ @khmakoto
 packages/react-image/ @cxe-prg
 packages/react-flex/ @cxe-prg
+packages/react-storybook/ @cxe-prg @ling1726 @layershifter
+packages/storybook/ @cxe-prg @ling1726 @layershifter
```

Comment:
Should both of these (or at least the old one) also have the build team as an owner?

Reply:
I don't see such a need ( I'd like to have our team and teams fluent team here for now ).
both are private for now, would be nice to shape up the new one to be a proper addon etc - then we can make it public.